src/openrct2-ui/windows/Finances.cpp

```diff
@@ -772,6 +772,8 @@ class FinancesWindow final : public Window
 
     void OnPrepareDrawMarketing()
     {
+        disabled_widgets = 0;
```

Comment:
I’d prefer to do this in OnPrepareDraw(). As it stands, the summary tabs don’t _happen_ to need it, but you can just assume that. Seems more robust to me to do it there.

Reply:
I see you already updated it now. Neither place is really good, it's a bit odd that this has to be updated inside `OnPrepareDraw` at all, but we don't have a proper system to handle it now. Ideally, the game would trigger an event that the window listens to, to update which widgets are enabled or disabled, instead of calculating them every time it gets drawn.